dev-support/docker/deploy.sh

```diff
@@ -0,0 +1,156 @@
+#!/bin/bash
+
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+usage() {
+    cat <<EOF 1>&2
+Usage: $0 [--help] [--hadoop <hadoop version>] [--tez <tez version>] [--hive <hive version>] [--repo <docker repo>]
+          [--hiveserver2 <start standalone HiveServer2>] [--metastore <start standalone Metastore>]
+Build the Hive Docker image and Run services
+--help           Help
+--hadoop         Hadoop version for building image
+--tez            Tez version for building image
+--hive           Hive version for building image
+--repo           Docker repository
```

Comment:
What does repo signify here?

Reply:
It's used to tag and push the image to remote hub repository